spec/features/ticket_purchases_spec.rb

```diff
@@ -108,6 +109,49 @@ def make_failed_stripe_purchase
         expect(purchase.paid).to be true
       end
 
+      scenario 'purchases a free registartion ticket' do
+        visit root_path
```

Comment:
line 113-120 and line 132-139 are identical. Since it's necessary for both of the tests, but not necessarily part of the test itself, you could extract these lines into a setup for these tests and then could also group those tests together.